stdlib/Pkg/src/Pkg.jl

```diff
@@ -13,9 +13,9 @@ Please see the manual section on packages for more information.
 """
 module Pkg
 
-export Dir, Types, Reqs, Cache, Read, Query, Resolve, Write, Entry
-export dir, init, rm, add, available, installed, status, clone, checkout,
-       update, resolve, test, build, free, pin, PkgError, setprotocol!
+#export Dir, Types, Reqs, Cache, Read, Query, Resolve, Write, Entry
+#export dir, init, rm, add, available, installed, status, clone, checkout,
+#       update, resolve, test, build, free, pin, PkgError, setprotocol!
```

Comment:
Should these lines be deleted?

Reply:
You accidentally reviewed #25705 as well, sorry about that I should probably have pointed the PR at that branch to minimise the diff.
cc @KristofferC